src/views/Dashboard.vue

```diff
@@ -1,13 +1,31 @@
 <template>
   <div class="about">
     <div class="my-information">
-      <p>{{ this.$store.getters.signInUser.displayName }}さんようこそ！！</p>
-      <p>残高:{{ this.$store.getters.myBalance }}</p>
+      <div>
+        <p class="block">{{ this.$store.getters.signInUser.displayName }}さんようこそ！！</p>
+      </div>
+      <div>
+        <p class="block">残高:{{ this.$store.getters.myBalance }}</p>
+        <button type="button" @click="signOut" class="btn btn-primary ml-2">ログアウト</button>
+      </div>
     </div>
     <h1 class='mt-5'>ユーザ一覧</h1>
   </div>
 </template>
 
+<script>
+export default {
+  methods: {
+    signOut() {
+      this.$store.dispatch('signOut')
+      .then(() => {
```

Comment:
thenの記述はインデントをつけましょう。
```suggestion
      this.$store.dispatch('signOut')
        .then(() => {
```